src/pages/SonarFeed/SonarFeedPage.js

```diff
@@ -89,25 +99,18 @@ const SonarFeed = ({ location: { pathname }, isLoggedIn, match }) => {
           <Route
             path={`${baseLocation}/details/:id`}
             exact
-            component={Loadable({
-              loader: () => import('./SignalDetails'),
-              loading: () => <PageLoader />
-            })}
+            render={props => <LoadableSignalDetailsPage {...props} />}
```

Comment:
In such case, when we don't need to pass specific props to a renderable route, we can pass desired component to a `Route` via `component` props. e.g: `component={LoadableSignalDetailsPage}`